gas/testsuite/gas/arc/taux.d

```diff
@@ -16,6 +16,7 @@
 .* U d2l
 .* U dpfp_status
 .* U fp_status
+.* U jli_base
```

Comment:
We can make a regex to handle this. We will fix it later. BTW, we need so jli tests as well. Please can u prepare/make some of them.